src/main/java/frc/robot/subsystems/StorageSubsystem.java

```diff
@@ -14,17 +14,17 @@
 import frc.robot.Constants.StorageConstants;
 
 public class StorageSubsystem extends SubsystemBase {
-  private WPI_TalonSRX conveyor = new WPI_TalonSRX(StorageConstants.kConveyorMotorPort);
-  private WPI_TalonSRX feeder = new WPI_TalonSRX(StorageConstants.kFeederMotorPort);
+  private WPI_TalonSRX this.conveyor = new WPI_TalonSRX(StorageConstants.kConveyorMotorPort);
+  private WPI_TalonSRX this.feeder = new WPI_TalonSRX(StorageConstants.kFeederMotorPort);
 
-  private boolean storageOrFeeder = true;
-  private boolean storageRunning = false;
+  private boolean this.storageOrFeeder = true;
+  private boolean this.storageRunning = false;
```

Comment:
great work on this too! however again, since the object is not fully declared, you can leave it as like "private WPI_TalonSRX conveyor = new WPI_TalonSRX(StorageConstants.kConveyorMotorPort" instead of using "this.conveyor"